math/subset_sum/info.toml

```
@@ -0,0 +1,23 @@
+title = 'Subset Sum'
+timelimit = 10.0
+forum = "https://github.com/yosupo06/library-checker-problems/issues/385"
+
+[[tests]]
+    name = "example.cpp"
+    number = 1
+[[tests]]
+    name = "big.cpp"
+    number = 5
+[[tests]]
+    name = "small.cpp"
+    number = 5
+[[tests]]
+    name = "random.cpp"
+    number = 5
+[[tests]]
+    name = "many_sets.cpp"
+    number = 5
+
+[params]
+    N_MAX=1000000
+    T_MAX=500000
```

```suggestion
    T_MAX=500000
    MOD=998244353
```
paramにしておくと問題文中で3桁区切りのカンマとかが自動で入って、便利です